corehq/apps/reports/util.py

```diff
@@ -346,6 +347,10 @@ def get_possible_reports(domain_name):
                     'path': model.__module__ + '.' + model.__name__,
                     'name': model.name
                 })
+
+    for slug, name, is_visible in get_extra_permissions():
+        if is_visible(domain):
+            reports.append({'path': slug, 'name': name})
```

Comment:
This is the main permissions change. By adding this to `get_possible_reports`, the 3 exports permissions will display to the user in the per-report permissions section. Behind the scenes, they're still being saved with the same path as before, even though there's no longer a report at that path: we're just treating it as a string constant—it never really mattered that it corresponded to the path, that just set up a easy mapping when I first wrote it. (And yes, I regret writing it that way back in the day.)